lago/cmd.py

```diff
@@ -623,23 +636,110 @@ def create_parser(cli_plugins, out_plugins):
-        help='Path to the workdir to use',
+        help='Path to the workdir to use.',
     )
     parser.add_argument(
         '--prefix-name',
         '-P',
         action='store',
         default='current',
         dest='prefix_name',
-        help='Name of the prefix to use',
+        help='Name of the prefix to use.',
+    )
+    parser.add_argument(
+        '--ssh-user',
+        action='store',
+        default='root',
+        help='User for SSH provider.',
+    )
+    parser.add_argument(
+        '--ssh-password',
+        action='store',
+        default='123456',
+        help='Password for SSH provider.',
+    )
+    parser.add_argument(
+        '--ssh-tries',
+        action='store',
+        default=100,
+        type=int,
+        help='Number of ssh time outs to wait before failing.',
+    )
+    parser.add_argument(
+        '--ssh-timeout',
+        action='store',
+        default=10,
+        type=int,
+        help='Seconds to wait before marking SSH connection as failed.'
+    )
+    parser.add_argument(
+        '--libvirt_url',
+        action='store',
+        default='qemu:///system',
+        help='libvirt URI, currently only '
+        'system'
+        ' is supported.'
     )
-    parser.add_argument('--ignore-warnings', action='store_true', )
+    parser.add_argument(
+        '--libvirt-user',
+        action='store',
+        help='libvirt user',
+    )
+    parser.add_argument(
+        '--libvirt-password',
+        action='store',
+        help='libvirt password',
+    )
+    parser.add_argument(
+        '--default_vm_type',
+        action='store',
+        default='default',
+        help='Default vm type',
+    )
+
+    parser.add_argument(
+        '--default_vm_provider',
+        action='store',
+        default='local-libvirt',
+        help='Default vm provider',
+    )
+    parser.add_argument(
+        '--default_root_password',
+        action='store',
+        default='123456',
+        help='Default root password',
+    )
+    parser.add_argument(
+        '--lease_dir',
+        action='store',
+        default='/var/lib/lago/subnets',
+        help='Path to store created subnets configurations'
+    )
+    parser.add_argument(
+        '--reposync-dir',
+        action='store',
+        default='/var/lib/lago/reposync',
+        help='Reposync dir if used',
+    )
+    parser.add_argument(
+        '--reposync-config',
+        help='Reposync config',
+        default='config.repo',
+        action='store',
+    )
+
+    parser.add_argument('--ignore-warnings', action='store_true')
+    parser.set_defaults(**config.get_section('lago', {}))
     verbs_parser = parser.add_subparsers(dest='verb', metavar='VERB')
     for cli_plugin_name, cli_plugin in cli_plugins.items():
         plugin_parser = verbs_parser.add_parser(
-            cli_plugin_name, **cli_plugin.init_args
+            name=cli_plugin_name,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
```

Comment:
Nope, reopend https://github.com/lago-project/lago/issues/98, I misread the title, ``lago shell --help`` is still broken ;/


``ArgumentDefaultsHelpFormatter`` - prints loaded defaults in help commands.